lib/src/lib/tokenizer/tokenizer.ts

```diff
@@ -36,7 +36,7 @@ export abstract class Tokenizer {
    * @param text The buffer to stringify
    */
   public tokenize(text: string): string {
-    return Array.of(...this.generateTokens(text)).join();
+    return Array.of(this.generateTokens(text)).join();
```

Comment:
```suggestion
    return this.generateTokens(text).join();
```

`generateTokens` already returns an array, so no need to make a new one.